clang/test/CIR/CodeGen/attribute-annotate-multiple.cpp

```diff
@@ -0,0 +1,73 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-cir -mmlir --mlir-print-ir-before=cir-lowering-prepare %s -o %t.cir 2>&1 | FileCheck %s -check-prefix=BEFORE
```

Comment:
Please also add a .cir file with very simple examples that show those annotations can be correctly parsed out of globals and functions.